lib/tasks/bundlers/generateComponentPreload.js

```diff
@@ -85,12 +85,25 @@ module.exports = function({
 				allNamespaces.forEach((ns) => {
 					if (ns !== namespace && ns.startsWith(`${namespace}/`)) {
 						filters.push(`!${ns}/`);
+						// Explicitly exclude manifest.json files of subcomponents since the general exclude above this
+						// comment only applies to the configured default file types, which do not include ".json"
+						filters.push(`!${ns}/**/manifest.json`);
```

Comment:
Not sure whether this is our intention. But I guess it's a good solution for the time bing.

The exclude-regex generated by ResourceFilterList always applies the default file types and therefore won't exclude `manifest.json`:
```js
// !library/h/components/subcomponent3/ --> exclude: 
/^library\/h\/components\/subcomponent3\/(?:[^/]+\/)*[^/]*(?:\.js|\.control\.xml|\.fragment\.html|\.fragment\.json|\.fragment\.xml|\.view\.html|\.view\.json|\.view\.xml|\.properties)$/
```